.travis.yml

```diff
@@ -118,6 +118,12 @@ jobs:
     - name: "Cleanup integration tests"
       stage: cleanup integration tests
       if: commit_message !~ \[skip-integration-tests\] OR branch = master
+      install:
+        - ./travis-ci/fetch-cache.sh
+        - npm install
```

Comment:
I'm not sure I understand why we're installing modules as part of the cleanup step?

Reply:
Ah, is it so we can use the `concurrency.mutex` from the common package?